python/dolfinx_contact/parallel_mesh_ghosting.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (C) 2022 Chris N. Richardson and Sarah Roggendorf
+#
+# SPDX-License-Identifier:    MIT
+
+from dolfinx.mesh import create_mesh, meshtags
+import dolfinx
+from dolfinx.cpp.mesh import entities_to_geometry, cell_num_vertices, cell_entity_type, to_type
+import numpy as np
+
+__all__ = ["create_contact_mesh"]
+
+
+def create_contact_mesh(mesh, fmarker, dmarker, tags):
```

Comment:
Some documentation of what happens to the ghosting of the mesh here would be useful.
Do we ghost all cells that are connected to facets of the opposite tag?